features/mbedtls/platform/TARGET_PSA/COMPONENT_PSA_SRV_IMPL/src/default_random_seed.cpp

```diff
@@ -0,0 +1,17 @@
+#include "mbed.h"
+#include "crypto.h"
+#include "default_random_seed.h"
+#include "psa_prot_internal_storage.h"
+
+int mbed_default_seed_read(unsigned char *buf, size_t buf_len)
+{
+    psa_its_status_t rc = psa_its_get(PSA_CRYPTO_ITS_RANDOM_SEED_UID, 0, buf_len, buf);
+    return ( -1 * rc );
```

Comment:
Minor: `- rc` would do as well, and a comment to explain why (which existed in a previous commit: ITS errors are positive but mbedos errors are negative) would be useful.

Reply:
@gilles-peskine-arm Fixed